pages/common/weasyprint.md

```diff
@@ -0,0 +1,23 @@
+# weasyprint
+
+> Render HTML to PDF or PNG.
+
+- Render a HTML file to PDF:
+
+`weasyprint {{path/to/input.html}} {{path/to/output}}.pdf`
+
+- Render a HTML file to PNG, including an additional user stylesheet:
+
+`weasyprint {{path/to/input.html}} {{path/to/output}}.png --stylesheet {{path/to/stylesheet.css}}`
+
+- Output additional debugging information when rendering:
+
+`weasyprint {{path/to/input.html}} {{path/to/output}}.pdf --verbose`
+
+- Specify a custom resolution when outputting to PNG:
+
+`weasyprint {{path/to/input.html}} {{path/to/output}}.png --resolution {{300}}`
```

Comment:
Does 300 mean 300x300 ? Can't we specify custom height and width ?

Reply:
It's in DPI I think.